light/rpc/client.go

```diff
@@ -409,7 +411,16 @@ func (c *Client) BlockResults(ctx context.Context, height *int64) (*ctypes.Resul
 
 func (c *Client) Commit(ctx context.Context, height *int64) (*ctypes.ResultCommit, error) {
 	// Update the light client if we're behind and retrieve the light block at the requested height
-	l, err := c.updateLightClientIfNeededTo(ctx, *height)
+	// or at the latest height if no height is provided.
+	var (
+		l   *types.LightBlock
+		err error
+	)
+	if height == nil {
+		l, err = c.lc.Update(ctx, time.Now())
+	} else {
+		l, err = c.updateLightClientIfNeededTo(ctx, *height)
```

Comment:
should we modify updateLightClientIfNeededTo to accept `nil` height and call Update?

Reply:
yeah I was thinking that. Perhaps that would be better